### ISSUE TITLE
[iOS] open shared location in

### ISSUE THREAD

**Issue by leonardpitzu** [organicmaps/organicmaps#3534]
I just have discovered OM and find it simply amazing. One thing missing though is the "open shared location in" menu. For example when i get a link to a shared location in Whatsapp i can select which app to use to navigate there - Google Maps, Apple Maps etc. OM is missing and it would be great to have it there.

**Comment by biodranik** [comment]
What kind of links do you want to open with OM? Shouldn't they be supported by Whatsapp?

**Comment by rtsisyk** [comment]
Yeah, the same problem with Telegram and other apps. My understanding the list of apps is just hard-coded in WhatsApp/Telegram.

**Comment by biodranik** [comment]
@rtsisyk can you (or someone else) please confirm that it works on Android in Telegram and Whatsapp?

**Comment by biodranik** [comment]
As far as I understand, it is implemented by WhatsApp and Telegram developers, right? Is it possible to contribute a PR into the Telegram source code that will display OM in the list of offered apps? Is it possible to report/ask WhatsApp developers for a similar feature?

**Comment by kirylkaveryn** [comment]
> As far as I understand, it is implemented by WhatsApp and Telegram developers, right? Is it possible to contribute a PR into the Telegram source code that will display OM in the list of offered apps? Is it possible to report/ask WhatsApp developers for a similar feature?

At least we can try to do that.
